src/GraphBLAS-sharp.Backend/Common/PrefixSum.fs

```diff
@@ -209,6 +209,7 @@ module PrefixSum =
     let runBackwardsIncludeInPlace plus = runInPlace plus true scanInclusive
 
     /// <summary>
+    /// This method is deprecated due to bad perfomance.
```

Comment:
Can we use [obsolete attribute](https://learn.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-8.0) here?